src/deploy/functions/build.ts

```diff
@@ -170,6 +190,10 @@ type Endpoint = Triggered & {
   //  process.env.FIREBASE_FUNCTIONS_DEFAULT_REGION
   region?: string[];
 
+  project: string;
```

Comment:
I don't feel any more at ease with these fields than you do. I'm perfectly happy to remove them from the Build if you want. Totally up to your discretion.